core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -193,7 +193,7 @@ public Result setOpToSql(SqlSetOperator operator, RelNode rel) {
    * @param leftFieldCount  Number of fields on left result
    * @return SqlNode that represents the condition
    */
-  public static SqlNode convertConditionToSqlNode(RexNode node,
+  public SqlNode convertConditionToSqlNode(RexNode node,
```

Comment:
why remove `static`?

Reply:
Sorry for the late reply. I have added dialect parameter in stripCastFromString method. As this method was static I could not use instance variable dialect to call stripCastFromString.

Reply:
I would prefer to still make it static because it is a tool method, you can add a SqlDialect as additional argument.

Reply:
I have modified my code and added SqlDialect as additional argument as you suggested. Thank you.